src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredMen = people
+    .filter(person => {
+      const { sex, died } = person;
+
+      return sex === 'm'
+      && (century
+        ? Math.ceil(died / 100) === century
+        : true);
+    })
+    .map(person => person.died - person.born);
```

Comment:
let's split this to two variables and assign this map to a different variable (`menAges` for example) as it will return just ages and it won't be suitable for `filteredMen` name anymore.